env0/resource_environment.go

```diff
@@ -20,6 +20,84 @@ func isTemplateless(d *schema.ResourceData) bool {
 }
 
 func resourceEnvironment() *schema.Resource {
+	configurationSchema := &schema.Resource{
```

Comment:
Moved the configuration schema to a variable.
Both the environment and the subenvironment may use the same schema.